### ISSUE TITLE
Hosts with static IPs are not read by dnsmasq.ReadLeases()

### ISSUE THREAD

**Issue by f18m** [b0ch3nski/go-dnsmasq-utils#1]
Hey @b0ch3nski ,
first of all thanks for this package. It saved me quite some time!

I noticed a missing feature though. 
You can start dnsmasq with an option like `--dhcp-host=e8:fd:f8:33:4f:80,appliance1,192.168.1.11,infinite` to indicate that you want to reserve a specific IP (perhaps outside the DHCP range) to a particular MAC address and the DHCP lease time should  be infinite.
In such case the dnsmasq.leases file will look like:

```
...
1726810188 80:64:6f:c7:f3:b8 192.168.1.188 shellyplus1-80646fc7f3b8 01:80:64:6f:c7:f3:b8
0 e8:fd:f8:33:4f:80 192.168.1.11 appliance1 01:e8:fd:f8:33:4f:80
...
```

The issue is that the regex currently used 

```
var rgxLease = regexp.MustCompile(`(\d{10}) (\S+) (\S+) (\S+) \S+`)
```

requires exactly ten digits as first field. Static hosts can have a zero as first field as in my example.
Can we accomodate that?

thanks!

**Comment by b0ch3nski** [comment]
Hi @f18m,
Glad you found this useful :slightly_smiling_face:
Please try #2 - I'd appreciate a feedback on this one as I'm on a train now with limited possibilities to test this on a real dnsmasq.

**Comment by f18m** [comment]
Works! thanks for the super-fast help

**Comment by b0ch3nski** [comment]
Thanks for testing! I've merged #2 and released as v0.1.1
